pkgs/development/python-modules/url-normalize/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry
+, pytest-cov
+, pytest-flakes
+, pytest-mock
+, pytest-socket
+, pytestCheckHook
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "url-normalize";
+  version = "1.4.3";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
```

Comment:
Why not `fetchPypi`? I case that applies, you should add a comment that tests are missing.

Reply:
Not a lot of packages have that and I wouldn't nit people about it. So it is nice to have but not hard required.